src/commons/services/utils.js

```diff
@@ -96,6 +96,24 @@
             });
         };
 
+        vm.importCluster = function(uri, data) {
+            var url, actionRequest, request;
+
+            url = config.baseUrl + uri;
+
+            actionRequest = {
```

Comment:
@cloudbehl Ack , valid point , will update in next PR . actually this PR got merged .